PermMissingElem/Solution.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace PermMissingElem;
+
+public class SolutionClass
+{
+    public int Solution(int[] A)
+    {
+        int s = 0;
```

Comment:
name variabile more explicit
In this case I can't see what this variabile is for? what is doing? Is it just a counter or a temp variabile